src/Target.php

```diff
@@ -254,28 +263,50 @@ public function getMessagePrefix(array $message): string
      * as follows:
      *
      * ```php
-     * 'enabled' => function() {
-     *     return !Yii::getApp()->user->isGuest;
+     * 'enabled' => function(\Yiisoft\Yii\Web\User $user) {
```

Comment:
It does not seems to work in this way. How `$user` will be injected to this function?

Reply:
"brain hiccup", thank you for catching it